Packs/HackerOne/pack_metadata.json

```diff
@@ -10,14 +10,9 @@
     "categories": [
         "Vulnerability Management"
     ],
-    "tags": [
```

Comment:
Why are these removed?

Reply:
We are revamping the entire tags and use cases sections for the XSOAR and XSIAM Marketplaces, as they have had too many options that have cluttered the search options.